kaal-presentation/src/main/kotlin/cz/eman/kaal/presentation/fragment/FragmentExtensions.kt

```diff
@@ -22,3 +26,17 @@ inline fun <T : Fragment> T.withArgs(argsBuilder: Bundle.() -> Unit): T = this.a
  * @since 0.2.0
  */
 inline fun <reified T : View> Fragment.findOptional(id: Int): T? = view?.findViewById(id) as? T
+
+/**
+ * Shows a keyboard using activity from fragment.
+ *
+ * @since 0.9.0
+ */
+fun Fragment.showKeyboard() {
+    (activity?.getSystemService(Context.INPUT_METHOD_SERVICE) as? InputMethodManager)?.apply {
```

Comment:
1. The `apply { ... }` block is not needed.
2. Use `as InputMethodManager?` instead of `as? InputMethodManager`